app/controllers/access_requests_controller.rb

```diff
@@ -8,11 +8,14 @@ def self.feature_enabled?
   def new
     session[:access_request_back_to] ||= request.referer
     @projects = Project.all.order(:name)
+    @roles = Role.all[1...-1] # ignore viewer (default) and super_admin (no such role per project)
```

Comment:
If here what you want is the Project Roles, then instead of `Role` you'll need to use `ProjectRole`. If you use `ProjectRole`, you won't need to use the range to filter out `Viewer` and `Supe Admin`.